### PR TITLE
Small fix default.handlebars move Logout-Bar-Style to style.css

### PR DESCRIPTION
**Pull request by JSuenram**
Just making a tiny first step to check if we can help you in some places bringing this great project forward....

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -308,3 +308,4 @@ __pycache__/
 *.btm.cs
 *.odx.cs
 *.xsd.cs
+.DS_Store
```
Comment (JSuenram) [inline .gitignore:311]: MacOS creates this file in every folder. Is not needed at all, so no need to sync.

**Comment by Ylianst** [comment]
Thank you! All help is very appreciated! After this pull request, the web pages don't seem to look correct, investigating what is going on now.

image

**Comment by JSuenram** [comment]
> Thank you! All help is very appreciated! After this pull request, the web pages don't seem to look correct, investigating what is going on now.
> 
> image

Will look into it. Maybe a small CSS-Glitch....

**Comment by Ylianst** [comment]
Also, if you hit the small diamond on the top right and select different view modes, they are complete broken.

image

I will revert some of the changes until fixed.

**Comment by JSuenram** [comment]
> Also, if you hit the small diamond on the top right and select different view modes, they are complete broken.
> 
> image
> 
> I will revert some of the changes until fixed.

I am sorry. I believe made a mistake in default.handlebars line 287 by adding a "title"-tag. Investigating also...

**Comment by Ylianst** [comment]
The title tag is fine, the style.css changes broke the page. I fixed it.

**Comment by JSuenram** [comment]
> The title tag is fine, the style.css changes broke the page. I fixed it.

I am sorry, have to be more careful in pre-stage-testing.... realy my fault.... sorry.....

**Pull request merged**